### DISCUSSION TITLE
Manage work profile?

### DISCUSSION THREAD

**Discussion by GreenMystic** [BinTianqi/OwnDroid#55] [Q&A] [answered]
hi developer I'm new to this app can 
Owndroid manage app on work profile as well?
(Ex: deny permission by default on work profile even app installed on main profile)

thanks for reading and giving answer developer 🙏

**Comment by BinTianqi** [comment, marked as answer]
If you want to manage apps in work profile, make sure that the work profile was created by OwnDroid, then you can use App manager of OwnDroid in your work profile.

Note that work profiles and primary user are independent of each other. In other words, OwnDroid in work profile cannot manage primary user, and OwnDroid in primary user cannot manage existing work profile.

**Reply by GreenMystic** [reply]
bruh 💀
that's mean i had to recreate work profile
and how i can clone app to work profile via owndroid?

I'm dead 💀

**Reply by BinTianqi** [reply]
What's your current work profile owner app? Island ? OwnDroid isn’t perfect at managing work profile, apps such as [Island](https://github.com/oasisfeng/island) maybe better.

**Reply by GreenMystic** [reply]
shelter
... I'm lazy to use island / insular atm
obviously I don't need their hack feature for now 🗿
